ui/src/components/header/UserProfile.tsx

```diff
@@ -52,14 +52,28 @@ export default function UserProfile(props: UserProfileProps) {
           metadata[authMethodIssuerKey as keyof typeof metadata];
         logoutURL = `//${logoutURI}`;
       }
+
+      const redirectAddressKey = 'io.flipt.auth.redirect_address';
+      if (metadata[redirectAddressKey as keyof typeof metadata]) {
+        let redirectAddress = metadata[redirectAddressKey as keyof typeof metadata] as string;
+        redirectAddress = redirectAddress?.trim();
```

Comment:
if you run `mage ui:fmt` it will fix any ui formatting issues for you automatically